src/cli/api/trackingplans.ts

```diff
@@ -121,15 +124,31 @@ export function parseTrackingPlanName(name: string): { id: string; workspaceSlug
   return {
     id,
     workspaceSlug,
+    APIVersion: 'v1',
   };
 }
 
-export function toTrackingPlanURL(name: string): string {
-  const { id } = parseTrackingPlanName(name);
-  return `https://api.rudderstack.com/trackingplans/${id}`;
+export function toTrackingPlanURL(trackingPlan: RudderAPI.TrackingPlan): string {
+  if (!trackingPlan.creationType) {
+    const { id } = parseTrackingPlanName(trackingPlan.name);
+    return `https://api.rudderstack.com/trackingplans/${id}`;
+  }
+  return `https://api.rudderstack.com/tracking-plans/${trackingPlan.id}`;
```

Comment:
```suggestion
  return `https://api.rudderstack.com/tracking-plans/v2/${trackingPlan.id}`;
```

Reply:
We do not have `v2` in the API signature
https://github.com/rudderlabs/rudder-config-backend/blob/master/src/modules/tracking-plans/apiGateway/routes.ts#L30